packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/encryption.js

```diff
@@ -143,6 +143,19 @@ describe('plugin-conversation', () => {
           }));
       });
 
+      describe('#cardAction()', () => {
+        it('creates a cardAction Activity', () => spark.internal.conversation.post(conversation, {
+          objectType: 'submit',
+          inputs: {key: 'value'}
+        })
+          .then(() => checkov.spark.internal.conversation.get(conversation, {activitiesLimit: 1}))
+          .then((c) => {
+            assert.property(c, 'defaultActivityEncryptionKeyUrl');
+            /* eslint-disable no-unused-expressions */
```

Comment:
Why disable?

Reply:
Linter is throwing an error for expect `no-unused-expression expected an assignment or function call`

Reply:
We'll have to address this globally because we are using chai more.